python/paddle/amp/auto_cast.py

```diff
@@ -247,7 +294,12 @@ def set_excluded_layers(models, excluded_layers):
                 layer._cast_to_low_precision = False
 
 
-def _pir_apply(self, func, dtype, include_sublayers=True):
+def _pir_apply(
+    self,
```

Comment:
```suggestion
    self: Layer,
```

`_pir_apply` 不在类的定义里面，所以这里要标注 `self` ～

另外，结合后面的调用关系 

``` python
                _pir_to_impl(
                    layer,
```

类型貌似是 `Layer`